app/helpers/layout_helper.rb

```diff
@@ -493,13 +498,23 @@ def new_child_fields_template(form_builder, association, options = { })
     end
   end
 
-  def spinner
-    '<span class="spinner-placeholder" />'.html_safe
-  end
-
   private
 
   def table_css_classes(classes = '')
     "table table-bordered table-striped table-condensed " + classes
   end
+
+  def submit_or_cancel(f, overwrite = false, args = { })
```

Comment:
can you add it back to its original location so it doesn't show in the diff? (its hard to tell if it was changed or not)